openassessment/workflow/models.py

```diff
@@ -179,6 +179,21 @@ def score(self):
         """
         return sub_api.get_latest_score_for_submission(self.submission_uuid)
 
+    @property
+    def override_score(self):
```

Comment:
How would you feel about renaming this, something like `score_overridden`? 

As is, (as a property defined with `def name` syntax and appearing to be of form `verb_noun`), this keeps looking to me like it's actually a method.